hybrasyl/World.cs

```diff
@@ -1407,6 +1407,8 @@ private void PacketHandler_0x07_PickupItem(Object obj, ClientPacket packet)
 
                     Logger.DebugFormat("Removing {0}, qty {1} from {2}@{3},{4}",
                         item.Name, item.Count, user.Map.Name, x, y);
+                    item.DeathPileTime = null;

```

Comment:
this seems unnecessary as the item is removed from the map directly below.

Reply:
In my testing, the DeathPileTime / Ownership continued after an item was picked up. If the user set it down after, these values continued.